test/test_checkout.py

```diff
@@ -0,0 +1,39 @@
+import os
+import subprocess
+
+import pytest
+
+def test_checkout(rename_git, git2cpp_path):
+    create_cmd = [git2cpp_path, 'branch', 'foregone']
+    subprocess.run(create_cmd, capture_output=True, cwd="test/data/status_data", text=True)
+
+    checkout_cmd = [git2cpp_path, 'checkout', 'foregone']
+    p = subprocess.run(checkout_cmd, capture_output=True, cwd="test/data/status_data", text=True)
+    assert(p.stdout == '');
```

Comment:
The python `assert` here doesn't actually need the brackets. But we already have a few of these, so we can just leave them until we have some linting set up on the test code and that will clean them up.